src/plugins/janus_streaming.c

```diff
@@ -548,6 +548,27 @@ multistream-test: {
 {
 	"streaming" : "ok"
 }
+\endverbatim
+ *
+ * You can kick all viewers from a mountpoint using the \c kick_all request. Notice
+ * that this only removes all viewers, but does not prevent them from starting to watch
+ * the mountpoint again. Please note this request works with all mountpoint types,
+ * except for on-demand streaming. The \c kick_all request has to be formatted as follows:
+ *
+\verbatim
+{
+	"request" : "kick_all",
+	"id" : <unique ID of the mountpoint to disable; mandatory>,
+	"secret" : "<mountpoint secret; mandatory if configured>",
+}
+\endverbatim
+ *
+ * If successful, a generic \c ok is returned:
+ *
+\verbatim
+{
+	"streaming" : "ok",
```

Comment:
I'd rather this be called "kicked", to be more in line with our response to, e.g., "create", "edit" and "destroy". I know we return "ok" in a few other synchronous requests, but I'm not fond of those, so for new requests we can try do something nicer :grin: